web/src/features/hfiCalculator/components/MeanPrepLevelCell.tsx

```diff
@@ -72,16 +72,27 @@ const MeanPrepLevelCell = (props: MeanPrepLevelCellProps) => {
     }
   }
 
+  const prepLevelError = () => {
```

Comment:
This can be simplified by just returning the conditional operation in the if statement

Reply:
Do you mean write the 3 conditional statements inside the classname?

Reply:
I mean `prepLevelError` can just be:

```
return  isUndefined(props.meanPrepLevel) ||
      isNaN(props.meanPrepLevel) ||
      props.emptyOrIncompleteForecast === true
```